js/jquery.mapael.js

```diff
@@ -55,6 +64,9 @@
                 , zoomCenterX = 0
                 , zoomCenterY = 0
                 , previousPinchDist = 0;
+                
+            // add plugin class name on element
+            $self.addClass(pluginName);
```

Comment:
About the second point, we will be able to define the CSS **for** mapael.
Example: the `.tooltip` class. There is high chance this name is already used on a website, or through a framework, with different parameters. 
Hence, using `.mapael .tooltip` will allow specific selectivity.